sherpa/conftest.py

```diff
@@ -117,7 +117,9 @@ def pytest_addoption(parser):
             [r"invalid value encountered in sqrt",
              # See https://github.com/ContinuumIO/anaconda-issues/issues/6678
              r"numpy.dtype size changed, may indicate binary " +
-             r"incompatibility. Expected 96, got 88"
+             r"incompatibility. Expected 96, got 88",
+             # See https://github.com/numpy/numpy/pull/432
+             r"numpy.ufunc size changed"
```

Comment:
So the ufunc change is *only* happening because we have a mis-match of NumPy versions on Travis? Why is that: are we building against NumPy version 1.x but some some other package we use has a (probably implicit) dependency on NumPy version 1.y (where y != x)? Do we know what package? I am wondering if we can set up travis so that we ensure a constant NumPy version (so hopefully not have to bother with white-listing this version), but maybe we don't want to go to that effort?

Reply:
As it is good practice, we build against an old version of numpy (1.11 seems to be a common choice) and then we test against a number of them. The warning is issued by Cython (See https://github.com/numpy/numpy/pull/432).

Most of our Travis builds have numpy pinned, while some have it float, so it's driven by other packages (astropy, in particular) and will be installed at its latest version. This "lookahead" jobs let any issues with the latest version of numpy surface as soon as possible, as it has happened in the past (e.g. when numpy changed the way they print arrays).

Reply:
Thanks for the clarification.